app/src/main/java/edu/uc/hickmadc/customnotifications/MainActivity.kt

```diff
@@ -1,38 +1,36 @@
 package edu.uc.hickmadc.customnotifications
 
-import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
 import android.widget.ListView
+import androidx.appcompat.app.AppCompatActivity
 import androidx.navigation.Navigation
 import androidx.navigation.findNavController
 import androidx.navigation.ui.NavigationUI
 import com.google.android.material.bottomnavigation.BottomNavigationView
 import edu.uc.hickmadc.customnotifications.ui.main.MyListAdapter
 
-//import edu.uc.hickmadc.customnotifications.ui.main.MainFragment
-
 class MainActivity : AppCompatActivity() {
 
-    //temp data
-    val language = arrayOf<String>("Brush Teeth","Eat Breakfast","Do schoolwork")
+    /*
+        Temporary data
+     */
+    val language = arrayOf<String>("Brush Teeth", "Eat Breakfast", "Do schoolwork")
     val description = arrayOf<String>(
         "Brush teeth for 2 minutes",
         "Eat so you don't feel like shit",
-        "")
```

Comment:
Having each of the descriptions match up with a language value is confusing if one of them is blank. IE: it is possible to say "wait why is it blank" and forget you had this hard coded string empty. So I just filled it.

Reply:
A String generic type is generally a sign of improper object oriented design.